tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/appsettings.json

```diff
@@ -212,23 +213,184 @@
     "ColumnNameModifiedDate": "Modified",
     "MatterConfigurationColumn": "ConfigurationValue",
     "SearchColumnsUIPickerForMatter": {
-      "Column1": "matterName",
-      "Column2": "matterClient",
-      "Column3": "matterClientId",
-      "Column4": "matterModifiedDate",
-      "Column5": "matterResponsibleAttorney",
-      "Column6": "matterSubAreaOfLaw",
-      "Column7": "matterCreatedDate"
+      "Column1": {

```

Comment:
Extended the SearchColumnsUIPickerForMatter json object to cater the needs of both project center and matter center when displaying the columns in the matter search grid